syntax = "proto3";

package Protocol;

message VECTOR3{
	float x = 1;
	float y = 2;
	float z = 3;
}